src/mavsdk/plugins/action_server/action_server_impl.h

```diff
@@ -75,16 +82,6 @@ class ActionServerImpl : public ServerPluginImplBase {
     std::atomic<bool> _force_disarmable = false;
     std::atomic<bool> _allow_takeoff = false;
 
-    union px4_custom_mode {
```

Comment:
Opportunistic cleanup, this declaration is a redundant copy of the one in `px4_custom_mode.h`